File: App/app/src/main/java/com/cinema/ticket_booking/CinemaApplication.java
package com.cinema.ticket_booking;

import com.cinema.ticket_booking.util.ThemeManager;
import dagger.hilt.android.HiltAndroidApp;

@HiltAndroidApp
public class CinemaApplication extends android.app.Application {
    @Override
    public void onCreate() {
        super.onCreate();
        // Apply saved theme preference ngay khi app khởi động
        ThemeManager.applyTheme(this);
    }
}

File: App/app/src/main/java/com/cinema/ticket_booking/CinemaApplication.kt
package com.cinema.ticket_booking

import android.app.Application
import com.cinema.ticket_booking.util.ThemeManager
import dagger.hilt.android.HiltAndroidApp

@HiltAndroidApp
class CinemaApplication : Application() {
    override fun onCreate() {
        super.onCreate()
        // Apply saved theme preference ngay khi app khởi động
        ThemeManager.applyTheme(this)
    }
}
